lib/sidebar.js

```diff
@@ -44,11 +44,13 @@ define(function () {
     };
 
     self.hide = function hide() {
-      container.classList.add('hidden');
+      container.children[1].classList.add('hide');
+      container.children[2].classList.add('hide');
     };
 
     self.reveal = function reveal() {
-      container.classList.remove('hidden');
+      container.children[1].classList.remove('hide');
+      container.children[2].classList.remove('hide');
```

Comment:
I do not like hard coded indexing.
Correct me if i am wrong.
Is it possible to get all children which are not infobox. `> *:not(.infobox)`

Of a for loop over all children starts at 1.
(I know it is also not beautiful)